### PR TITLE
Fixing font path for kawkab-mono

### PR DESCRIPTION
**Pull request by aziz**

### CODE REVIEW

File: Casks/font-kawkab-mono.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
   homepage 'http://makkuk.com/kawkab-mono'
   license :ofl
 
-  font 'KawkabMono-Bold.otf'
-  font 'KawkabMono-Light.otf'
-  font 'KawkabMono-Regular.otf'
+  font 'kawkab-mono-0.501/KawkabMono-Bold.otf'
```
Comment (vitorgalvao) [inline Casks/font-kawkab-mono.rb]: Use `#{version}` instead of `0.501`. Quotes will also need to be changed to double quotes.

**Comment by aziz** [comment]
@vitorgalvao fixed it

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by aziz** [comment]
@vitorgalvao done

**Pull request merged**